app/helpers/users_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module UsersHelper
```

Comment:
All strings in this file are frozen and can not be modified. https://stackoverflow.com/questions/37799296/what-does-the-comment-frozen-string-literal-true-do, rubocop complains otherwise